Detectors/CTP/workflowScalers/src/ctp-ccdb-orbit.cxx

```diff
@@ -76,7 +77,7 @@ int main(int argc, char** argv)
   std::vector<int64_t> vect;
   std::string ccdbPath;
   auto now = std::chrono::system_clock::now();
-  long tt = std::chrono::duration_cast<std::chrono::milliseconds>(now.time_since_epoch()).count();
```

Comment:
Is this a internal change of unit or will it effect information in CCDB objects?

Reply:
It is to be change of the ccdb object content. Now after fix (second commit) it should be ok.